cmd/config/cache/config.go

```diff
@@ -64,6 +69,15 @@ func (cfg *Config) UnmarshalJSON(data []byte) (err error) {
 	if _cfg.After < 0 {
 		return errors.New("cache after value should not be less than 0")
 	}
+	if _cfg.WatermarkLow < 0 || _cfg.WatermarkLow > 100 {
+		return errors.New("config low watermark value should be between 0 and 100")
+	}
+	if _cfg.WatermarkHigh < 0 || _cfg.WatermarkHigh > 100 {
+		return errors.New("config high watermark value should be between 0 and 100")
+	}
+	if _cfg.WatermarkLow > _cfg.WatermarkHigh {
```

Comment:
```suggestion
	if _cfg.WatermarkLow >= _cfg.WatermarkHigh {
```
At least then there will be a little headroom ;)

Reply:
fixed